src/CqlBuilderPanel/Includes/Results/IncludeResultActions.tsx

```diff
@@ -18,37 +23,66 @@ const IncludeResultActions = ({
   onEdit,
   onDelete,
   onView,
+  operation,
 }: PropTypes) => {
   return (
     <Stack direction="row" alignItems="center">
       {canEdit && (
         <>
           {showDeleteAction && (
```

Comment:
{canEdit && showDeleteAction && ()} --> I think both conditions can be simplified into one